apps/explorer/src/utils/stringUtils.ts

```diff
@@ -51,10 +51,12 @@ export function truncate(fullStr: string, strLen: number, separator?: string) {
 const ELLIPSIS = '\u{2026}';
 export function formatAddress(address: string) {
     const offset = address.startsWith('0x') ? 2 : 0;
+    const addressPrefix = offset ? '0x' : '';
```

Comment:
This is probably the wrong way to do this, probably should just call a different `formatTransaction` function or something from the internal link component.

Reply:
Done